dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-1.1/resources/l10n/com.ibm.websphere.appserver.mpFaultTolerance-1.1.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides support for the MicroProfile Fault Tolerance 1.1 API for enterprise Java.
```

Comment:
You must capitalize Enterprise Java.

Reply:
The FaultTolerance-1.0 has it non-capitalized as well. Should we go back and change it there too?

Reply:
Interestingly, mpConfig-x.x doesn't mention Enterprise Java and microprofile-1.3 says it's for Cloud Native Java.